### PR TITLE
Update zenith readmes:

### PR DESCRIPTION
**Pull request by lubennikovaav**
- Move source tree overview into separate README_sourcetree.md and update it.
- Move local installation guide into separate docs/install.md file.
- Add glossary: docs/glossary.md
- Add a draft of the Architecture overview to the main Readme.md

### CODE REVIEW

**Review by hlinnaka — APPROVED** [review]
Great, thanks!

File: README_sourcetree.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+## Source tree layout
```
Comment (ololobus) [inline README_sourcetree.md:1]: I would suggest moving this file to `docs/sourcetree.md` and reference it in the `docs/README.md` TOC

File: README.md [hunk 2]
```diff
@@ -1,160 +1,38 @@
 # Zenith
 
-Zenith substitutes PostgreSQL storage layer and redistributes data across a cluster of nodes
+Zenith substitutes PostgreSQL storage layer and redistributes data across a cluster of nodes.
 
-## Running local installation
+## Architecrure overview
 
-1. Install build dependencies and other useful packages
+A Zenith installation consists of Compute nodes, Storage engine and .
```
Comment (ololobus) [inline README.md:8]: For me it kinda helpful to have an installation overview in the base README.md. When you see a new project it helps a lot to start hacking

**Review by ololobus — APPROVED** [review]
Thanks for working on that!

**Pull request merged**